content/english/python-tensorflow/activity-6.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "Activity 6 - Testing the Model"
+date: 2020-09-09T16:19:17-07:00
+weight: 9
+draft: false
+---
+
+Let's repeat the same thing with the second clothing sample you chose in Activity 2. Change the index value to the one used for the second image in Activity 2. Then, repeat Activity 5 starting from `predictions[your_index_value]`
```

Comment:
This activity might just be able to be merged with Activity 5.

Reply:
Considered